build.gradle.kts

```diff
@@ -47,6 +47,8 @@ tasks.register("preCommit") {
         exec {
             commandLine("ktlintCheck", "app/src/**/*.kt")
             commandLine("ktlintCheck", "core/**/*.kt")
+            // Uncomment this line to run linter on example folder
+            // commandLine("ktlintCheck", "example/**/*.kt")
```

Comment:
is possible to put this optional to let it uncommented?

Reply:
Is commented as default to avoid any exception when removing the example module